torch/_decomp/__init__.py

```diff
@@ -264,6 +264,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.aminmax,
             aten.arange.default,
             aten.arange.start,
+            aten.as_strided_copy,
```

Comment:
So, apparently inductor doesn't like `as_strided`, so we should not add it to this list. We then should remove it from this list (not decompose it, I guess). See if inductor handles as_strided at all, if it does, we should handle this one te same way than `as_strided`, if not, then we should just not decompose it and let inductor fallback to eager.

Reply:
I did manage to get the inductor tests to work for everything except `torch.bool`, which I could not puzzle out as to why it didn't work, so I `skip`ped it.

Reply:
Well, that was a false claim, it fails for all types. Not sure why I thought otherwise. Updating coming.